test-infra/auto-deploy/deploy-cron.yaml

```diff
@@ -13,7 +13,12 @@ spec:
         spec:
           containers:
           - name: deploy-worker
-            image: gcr.io/gabrielwen-learning/deploy-worker:latest
+            image: gcr.io/kubeflow-ci/deploy-worker:latest
```

Comment:
nit: I suggest pinning to a specific image by using a tag that won't get moved to a different image; the latest tag is typically mutable.

This way the cron job won't break just because someone builds a new image with latest.

Also, I prefer to use "command" as oppose to "args" in the pod template spec. I think this is better because it puts the full command line in one place rather than splitting it between the entrypoint defined in the dockerfile and the args soo replace this with

command:
  - /usr/local/bin/init.sh
  - /src
  - kubeflow
  ....

Reply:
Done.